app/models/manageiq/providers/nsxt/nsxt_client.rb

```diff
@@ -3,16 +3,16 @@
 require 'json'
 class ManageIQ::Providers::Nsxt::NsxtClient
   include Vmdb::Logging
-  def initialize(server, user, password)
+  def initialize(server, user, password, verify_ssl = false)
     @server = server
     @user = user
     @password = password
-    @client = Rest.new(server, user, password)
+    @client = Rest.new(server, user, password, verify_ssl)
     connected, data = @client.login
-    if connected
-      return
-    end
+    return if connected
+
     $nsxt_log.error('NSX-T Authentication failed')
+    raise 'NSX-T Authentication failed'
```

Comment:
@gubbe505 for the credential validation to work properly, I need to raise an exception if there's no successful connection. Are you okay with this change? Will this not break anything? Also maybe the logging line is not necessary then ... cc @agrare

Reply:
I'm okay with the change. And I agree the logging can be removed.